datasets/loader.py

```diff
@@ -1,4 +1,4 @@
-#===============================================================================
+# ===============================================================================

```

Comment:
Nice to split this file at least to:
- classification datasets
- regression datasets

Reply:
+, please refer to https://github.com/catboost/benchmarks/blob/master/training_speed/data_loader.py

It's nice to have abalone and letters as well in the benchmarks

Reply:
Split to 3 files (clf, reg, mult)
Added these two datasets also